conf/site-swarm.conf

```diff
@@ -0,0 +1,24 @@
+server {
+        listen                  80 default_server;
```

Comment:
Will you map all enviroments to this port only? We had a lot of configs for this... 🤔